source/_layouts/base.twig

```diff
@@ -66,7 +66,7 @@
                     <a class="site_menu__link {% if page.url matches '#^/psr#' %}site_menu__link--active{% endif %}" href="/psr">PSRs</a>
                 </li>
                 <li class="site_menu__item">
-                    <a class="site_menu__link {% if page.url matches '#^/per#' %}site_menu__link--active{% endif %}" href="/per">PERs</a>
+                    <a class="site_menu__link {% if page.url matches '#^/per#' and not page.url matches '#^/personnel#' %}site_menu__link--active{% endif %}" href="/per">PERs</a>
```

Comment:
It works, but in this way it breaks the PERs active class. It's probably a precedence issue, because in this way it works in both cases:
```suggestion
                    <a class="site_menu__link {% if page.url matches '#^/per#' and not (page.url matches '#^/personnel#') %}site_menu__link--active{% endif %}" href="/per">PERs</a>
```